### PR TITLE
creates file with email and probability phish

### PR DESCRIPTION
**Pull request by nexusapoorvacus**
This PR has code that generates a text file (email_probabilities.txt) in common/output folder when python spear_phishing_detector.py --all is run. There is one line per email with the email's message_ID and probability it is a phish. These values are comma separated. The file gets overwritten every time spear_phishing_detector.py is run and the file is created if it doesn't exist initially.

### CODE REVIEW

File: common/classify.py [hunk 1]
```diff
@@ -119,6 +119,9 @@ def test_and_report(self):
         if not os.path.exists(self.results_dir):
             os.makedirs(self.results_dir)
 
+        # creates this file in common/output
+        email_probabilities = open(os.path.join("output", "email_probabilities.txt"), "w")
```
Comment (MatthewChow) [inline common/classify.py:123]: We probably want to close this at the end when we're done writing everything.

**Review by MatthewChow — COMMENTED** [review]
Overall, looks good to me, minus 1 comment. If possible, could you also show us some sample lines of what this file will look like (maybe just the first 10 lines or so when you run it) as either a gist or just pasted into the summary of this PR?

Thanks! Feel free to merge this after addressing my comment about closing the file object.

File: common/classify.py (hunk 1 above)
Comment (nexusapoorvacus) [reply common/classify.py:123]: Good point. I will add this and merge.

**Comment by nexusapoorvacus** [comment]
Below are a couple lines of the file that is generated. The format is message_id,phish probability:

`<CAPRpNNyw0Q1GmibEkqrYv6fO31MYR3k=-iJ-wxQbzZ1hFASA4w@mail.gmail.com>,0.499852101198
<570c6b68.451f620a.e2142.ffffa94a@mx.google.com>,0.333431842361
<570c6b37.c35a420a.9eb8.ffffb062@mx.google.com>,0.333431842361
<570c6b43.0922620a.2d613.ffffb29d@mx.google.com>,0.333431842361
<570c6b42.4347620a.76bcc.ffffad39@mx.google.com>,0.333431842361
<570c6b41.88ad420a.427c0.ffffa719@mx.google.com>,0.333431842361
<570c6b40.4aa8420a.e79a7.ffffb235@mx.google.com>,0.333431842361
<570c6b3f.833e420a.a81af.ffff8111@mx.google.com>,0.333431842361
<570c6b3e.4149620a.50b7c.ffffa32d@mx.google.com>,0.333431842361`

**Pull request merged**